torch/utils/tensorboard/writer.py

```diff
@@ -664,15 +653,12 @@ def add_graph(self, model, input_to_model=None, verbose=False, **kwargs):
             )
             if isinstance(model, list):
                 if isinstance(model[0], core.Net):
-                    current_graph = nets_to_graph_def(
-                        model, **kwargs)
```

Comment:
There is no test for caffe's graph yet in pytorch repo, so I suggest leave it as is.

Reply:
In that case, let's add some tests before I land this. I'll file a separate task.

Reply:
Accidentally opened a new branch: https://github.com/pytorch/pytorch/tree/remove-tensorboard-experimental